CRITICAL - 2020-05-24 00:00:25 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(863): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 863, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(161): CodeIgniter\Model->update(Array, Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 00:34:27 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(179): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 00:36:43 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(180): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 00:39:35 --> syntax error, unexpected '=', expecting ']'
#0 C:\xampp\htdocs\dev\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\dev\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(865): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\dev\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-05-24 00:39:45 --> Undefined variable: primaryKey
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(180): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 180, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 00:48:57 --> Duplicate entry '22' for key 'PRIMARY'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `us...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2234): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(729): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\dev\system\Model.php(557): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\dev\app\Controllers\Users.php(181): CodeIgniter\Model->save(Array)
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#9 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-05-24 00:49:34 --> Duplicate entry '22' for key 'PRIMARY'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `us...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2234): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(729): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\dev\system\Model.php(557): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\dev\app\Controllers\Users.php(181): CodeIgniter\Model->save(Array)
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#9 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-05-24 01:24:34 --> syntax error, unexpected ')', expecting ']'
#0 C:\xampp\htdocs\dev\system\CodeIgniter.php(360): CodeIgniter\CodeIgniter->tryToRouteIt(NULL)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#2 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#3 {main}
CRITICAL - 2020-05-24 01:25:09 --> {0} filter must have a matching alias defined.
#0 C:\xampp\htdocs\dev\system\Filters\Filters.php(324): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('auth')
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(369): CodeIgniter\Filters\Filters->enableFilter('auth', 'before')
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-05-24 01:27:29 --> {0} filter must have a matching alias defined.
#0 C:\xampp\htdocs\dev\system\Filters\Filters.php(324): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('auth')
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(369): CodeIgniter\Filters\Filters->enableFilter('auth', 'before')
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-05-24 01:27:38 --> {0} filter must have a matching alias defined.
#0 C:\xampp\htdocs\dev\system\Filters\Filters.php(324): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('auth')
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(369): CodeIgniter\Filters\Filters->enableFilter('auth', 'before')
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-05-24 02:25:32 --> Duplicate entry '22' for key 'PRIMARY'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `us...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2234): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(729): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\dev\system\Model.php(557): CodeIgniter\Model->insert(Array, false)
#6 C:\xampp\htdocs\dev\app\Controllers\Users.php(181): CodeIgniter\Model->save(Array)
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#9 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-05-24 02:26:29 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(181): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 02:26:58 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(181): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 02:27:20 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(181): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 02:27:40 --> Undefined variable: primaryKey
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(180): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 180, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 02:27:58 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(180): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 02:30:59 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\app\Views\profile.php(10): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 10, Array)
#1 C:\xampp\htdocs\dev\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\dev\system\Common.php(1087): CodeIgniter\View\View->render('profile', Array, true)
#3 C:\xampp\htdocs\dev\app\Controllers\Users.php(202): view('profile')
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#5 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-05-24 02:31:58 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\app\Views\profile.php(10): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 10, Array)
#1 C:\xampp\htdocs\dev\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\dev\system\Common.php(1087): CodeIgniter\View\View->render('profile', Array, true)
#3 C:\xampp\htdocs\dev\app\Controllers\Users.php(202): view('profile')
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#5 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-05-24 02:32:21 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\app\Views\profile.php(10): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 10, Array)
#1 C:\xampp\htdocs\dev\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\dev\system\Common.php(1087): CodeIgniter\View\View->render('profile', Array, true)
#3 C:\xampp\htdocs\dev\app\Controllers\Users.php(202): view('profile')
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#5 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-05-24 02:42:50 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(863): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 863, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(181): CodeIgniter\Model->update(Array, Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 02:43:11 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(863): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 863, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(181): CodeIgniter\Model->update(Array, Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 02:43:28 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(863): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 863, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(181): CodeIgniter\Model->update(Array, Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 02:43:43 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(863): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 863, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(181): CodeIgniter\Model->update(Array, Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 02:50:07 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(180): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 03:00:52 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(182): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 03:01:09 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(182): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 03:02:13 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(182): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 03:14:11 --> Undefined property: App\Controllers\Users::$model
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(179): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 179, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 03:15:33 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(185): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 03:15:50 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(185): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 03:16:07 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(185): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 03:16:28 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(185): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 03:16:51 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(185): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 03:17:13 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(185): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 03:17:47 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(185): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 03:18:56 --> syntax error, unexpected '=', expecting ')'
#0 C:\xampp\htdocs\dev\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\dev\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(865): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\dev\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-05-24 03:25:18 --> Undefined variable: builder
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(190): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 190, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 03:25:30 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(191): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 03:26:02 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(191): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 03:26:37 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(191): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 03:26:55 --> Undefined variable: builder
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(194): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 194, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 03:27:44 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(191): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 10:07:02 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(183): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 10:07:38 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(183): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 10:07:57 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(183): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 10:11:35 --> syntax error, unexpected ';', expecting ')'
#0 C:\xampp\htdocs\dev\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\dev\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(865): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\dev\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-05-24 10:12:12 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(863): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 863, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(167): CodeIgniter\Model->update(Array, Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 10:13:09 --> Undefined variable: id
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(161): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 161, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 10:13:16 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(863): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 863, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(167): CodeIgniter\Model->update(Array, Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 10:26:14 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(728): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 728, Array)
#1 C:\xampp\htdocs\dev\system\Model.php(557): CodeIgniter\Model->insert(Array, false)
#2 C:\xampp\htdocs\dev\app\Controllers\Users.php(186): CodeIgniter\Model->save(Array)
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#5 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2020-05-24 10:31:52 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 10:32:11 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 10:34:13 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 10:37:07 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 10:37:26 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 10:37:50 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 10:40:35 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 10:46:20 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 10:46:42 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 10:48:03 --> Undefined variable: id
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(181): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 181, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 10:48:19 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 10:48:26 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 10:48:55 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 10:53:18 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 10:53:31 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(184): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 10:59:23 --> Undefined variable: builder
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(188): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 188, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 11:00:23 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(189): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 11:23:59 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(728): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 728, Array)
#1 C:\xampp\htdocs\dev\system\Model.php(557): CodeIgniter\Model->insert(Array, false)
#2 C:\xampp\htdocs\dev\app\Controllers\Users.php(165): CodeIgniter\Model->save(Array)
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#5 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2020-05-24 11:24:50 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(728): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 728, Array)
#1 C:\xampp\htdocs\dev\system\Model.php(557): CodeIgniter\Model->insert(Array, false)
#2 C:\xampp\htdocs\dev\app\Controllers\Users.php(165): CodeIgniter\Model->save(Array)
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#5 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2020-05-24 11:26:03 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(728): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 728, Array)
#1 C:\xampp\htdocs\dev\system\Model.php(557): CodeIgniter\Model->insert(Array, false)
#2 C:\xampp\htdocs\dev\app\Controllers\Users.php(170): CodeIgniter\Model->save(Array)
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#5 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2020-05-24 11:27:24 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(728): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 728, Array)
#1 C:\xampp\htdocs\dev\system\Model.php(557): CodeIgniter\Model->insert(Array, false)
#2 C:\xampp\htdocs\dev\app\Controllers\Users.php(170): CodeIgniter\Model->save(Array)
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#5 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2020-05-24 11:35:24 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(863): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 863, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(192): CodeIgniter\Model->update(Array, Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 11:36:08 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(863): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 863, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(192): CodeIgniter\Model->update(Array, Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 11:36:17 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(863): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 863, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(192): CodeIgniter\Model->update(Array, Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 11:40:30 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(863): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 863, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(193): CodeIgniter\Model->update(Array, Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 11:40:58 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\dev\system\Model.php(863): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 863, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(193): CodeIgniter\Model->update(Array, Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 11:44:54 --> There is no data to {0}.
#0 C:\xampp\htdocs\dev\system\Model.php(809): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(193): CodeIgniter\Model->update(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 11:49:51 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(182): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 11:51:45 --> Undefined index: idUsers
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(54): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 54, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(38): App\Controllers\Users->setUserSession(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->index()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 11:52:14 --> Unknown column 'idUsers' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(1845): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(489): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(209): CodeIgniter\Model->first()
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 11:54:00 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(182): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 11:55:29 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(182): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 11:56:29 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(182): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 11:58:14 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(182): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 11:59:37 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(187): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 12:00:52 --> Unknown column 'idUsers' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(1845): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(489): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(219): CodeIgniter\Model->first()
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 12:02:52 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(186): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 12:06:28 --> Unknown column 'idUsers' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(1845): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(489): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(219): CodeIgniter\Model->first()
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 12:19:35 --> Undefined variable: builder
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(185): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 185, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 12:26:51 --> Undefined variable: builder
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(185): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 185, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 12:27:48 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(187): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 12:29:43 --> Unknown column 'idUsers' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(1845): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(489): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(219): CodeIgniter\Model->first()
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 12:30:14 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(187): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 12:31:38 --> Unknown column 'idUsers' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(1845): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(489): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(219): CodeIgniter\Model->first()
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 12:32:34 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(187): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 12:33:28 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(187): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 12:34:40 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(187): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 12:35:04 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(187): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 12:35:45 --> Unknown column 'users.id' in 'where clause'
#0 C:\xampp\htdocs\dev\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `users` ...')
#1 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `users` ...')
#2 C:\xampp\htdocs\dev\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `users` ...')
#3 C:\xampp\htdocs\dev\system\Database\BaseBuilder.php(2441): CodeIgniter\Database\BaseConnection->query('UPDATE `users` ...', Array, false)
#4 C:\xampp\htdocs\dev\system\Model.php(864): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\dev\app\Controllers\Users.php(187): CodeIgniter\Model->update(Array, Array)
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->profile()
#7 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-05-24 12:43:16 --> Undefined index: idUsers
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(54): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 54, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(38): App\Controllers\Users->setUserSession(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->index()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 12:43:41 --> Undefined index: idUsers
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(54): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 54, Array)
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(38): App\Controllers\Users->setUserSession(Array)
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->index()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 12:55:07 --> Argument 3 passed to CodeIgniter\Router\RouteCollection::get() must be of the type array or null, string given, called in C:\xampp\htdocs\dev\app\Config\Routes.php on line 37
#0 C:\xampp\htdocs\dev\app\Config\Routes.php(37): CodeIgniter\Router\RouteCollection->get('dashboard', 'profile', 'Dashboard::prof...', Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(776): require('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(360): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 21:05:11 --> syntax error, unexpected 'echo' (T_ECHO), expecting ';' or ','
#0 C:\xampp\htdocs\dev\system\Common.php(1087): CodeIgniter\View\View->render('dashboard', Array, true)
#1 C:\xampp\htdocs\dev\app\Controllers\Dashboard.php(10): view('dashboard')
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Dashboard->index()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 21:20:42 --> Undefined variable: fname
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(93): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 93, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->register()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 21:39:15 --> Array to string conversion
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(94): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 94, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->register()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 21:53:39 --> Undefined variable: firstnameUnique
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(96): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 96, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->register()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 21:54:04 --> Array to string conversion
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(102): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 102, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->register()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 21:57:40 --> Array to string conversion
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(101): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 101, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->register()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 23:19:00 --> {0} is not a valid rule.
#0 C:\xampp\htdocs\dev\system\Controller.php(203): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('test1')
#1 C:\xampp\htdocs\dev\app\Controllers\Users.php(95): CodeIgniter\Controller->validate('test1')
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->register()
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-05-24 23:29:16 --> syntax error, unexpected '=>' (T_DOUBLE_ARROW), expecting ']'
#0 C:\xampp\htdocs\dev\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\dev\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(865): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\dev\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-05-24 23:43:36 --> syntax error, unexpected '=>' (T_DOUBLE_ARROW), expecting ']'
#0 C:\xampp\htdocs\dev\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\dev\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\dev\system\CodeIgniter.php(865): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\dev\system\CodeIgniter.php(390): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-05-24 23:44:01 --> Undefined variable: validation
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(90): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 90, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->register()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 23:44:26 --> Undefined variable: model
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(90): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 90, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->register()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-05-24 23:44:57 --> Call to a member function getError() on array
#0 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->register()
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-05-24 23:48:42 --> Undefined variable: error
#0 C:\xampp\htdocs\dev\app\Controllers\Users.php(101): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 101, Array)
#1 C:\xampp\htdocs\dev\system\CodeIgniter.php(914): App\Controllers\Users->register()
#2 C:\xampp\htdocs\dev\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 C:\xampp\htdocs\dev\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\dev\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
